core/themes/seven/css/vertical-tabs.css

```diff
@@ -34,16 +37,16 @@
   border-top: 1px solid #ccc;
   padding: 0;
 }
- @media (min-width: 750px) {
+@media (min-width: 750px) {
   .vertical-tabs-list {
     display:block;
     float: left; /* LTR */
     margin: -1px 0 -1px -15em; /* LTR */
   }
-   [dir="rtl"] .vertical-tabs-list {
-    margin-left: 0;
-    margin-right: -15em;
+  [dir="rtl"] .vertical-tabs-list {
     float: right;
+    margin: 0 -15em -1px 0;
+    border-top: none;
```

Comment:
Why did the top margin change from `-1px` to `0` here? It looks like setting `border-top: none` might balance that all out, but the rules shouldn't show/hide the border based on language direction.

Reply:
...oops 😅